.devcontainer/devcontainer.json

```diff
@@ -1,15 +1,19 @@
 {
-    "name": "Salesforce Development Container",
-    "image": "salesforce/salesforcedx:latest-slim",
-    "extensions": [
-      "salesforce.salesforcedx-vscode",
-      "redhat.vscode-xml",
-      "dbaeumer.vscode-eslint",
-      "esbenp.prettier-vscode",
-      "salesforce.codey-midnight"
-    ],
-    "settings": {
-      "salesforcedx-vscode-apex.java.home": "/usr/lib/jvm/java-11-openjdk-amd64",
-      "workbench.colorTheme": "Codey Midnight"
+  "name": "Salesforce Development Container",
+  "image": "salesforce/salesforcedx:latest-slim",
+  "extensions": [
+    "salesforce.salesforcedx-vscode",
+    "redhat.vscode-xml",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "salesforce.codey-midnight",
+    "/root/workspace/sfdx-env/.devcontainer/code-builder-vscode-0.0.4.vsix"
```

Comment:
The path to the extension needs to be an absolute path. I thought about shortening the name of the vsix file so it looks cleaner. I am open to that still but I found it helpful to know which version of the vsix was currently being used.